roles/openshift_cloud_provider/tasks/vsphere.yml

```diff
@@ -3,4 +3,26 @@
   template:
     dest: "{{ openshift.common.config_base }}/cloudprovider/vsphere.conf"
     src: vsphere.conf.j2
-  when: openshift_cloudprovider_vsphere_username is defined and openshift_cloudprovider_vsphere_password is defined and openshift_cloudprovider_vsphere_host is defined and openshift_cloudprovider_vsphere_datacenter is defined and openshift_cloudprovider_vsphere_datastore is defined
+  when: 
+  - openshift_cloudprovider_vsphere_username is defined 
+  - openshift_cloudprovider_vsphere_password is defined 
+  - openshift_cloudprovider_vsphere_host is defined 
+  - openshift_cloudprovider_vsphere_datacenter is defined 
+  - openshift_cloudprovider_vsphere_datastore is defined
+
+- name: configure vsphere svc account
+  include_role:
+    name: openshift_cloud_provider
+    tasks_from: vsphere-svc
```

Comment:
This could just be `import_tasks: vsphere-svc.yml` since we're alreay in openshift_cloud_provider role.